onboard/macos/cassandra.md

```diff
@@ -0,0 +1,19 @@
+# Temporal Server Cassandra Setup Guide
+
+## v3.11
+
+### Install
+```zsh
+brew install cassandra@3.11
```

Comment:
All commands that users are supposed to run manually are prefixed with `$` in other MD files. I would keep it.
```suggestion
$ brew install cassandra@3.11
```

Reply:
should the command be ready. meaning i can basically copy from the beginning of the line to the end of the line into the terminal and execute? additional `$` will cause more work

Reply:
Ya, we actually do NOT want `$` prefixed in commands. 
I am removing these wherever I find them.
> additional $ will cause more work
^ I agree with Wenquan

Reply:
I think `$` makes sense when you show example of command and it's output. Just to differentiate between command itself and output it produces. Here, you don't have any outputs, so it is just for consistency.

@flossypurse, can you remove ir from other MD files in this repo?

Reply:
Ya, I am happy to remove the `$` from other command examples in the repo.

My recommendation is to separate command input and command output like this:

Run this command:
```bash
go run main.go
```
Example output:
```bash
Hello World
```